Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,528 @@
+
+> If you do not need to back up or share your terraform state, you can remove the "backend" block and terraform will store it's state locally.
+
+Next, we need to set some specific-to-you configurations.
+
+Create a `vars.tfvars` file similar to the following:
+
+```
+aws_profile="catena_admin"                  # Your AWS Profile
+aws_region="us-east-1"                      # The region you are working in
+ec2_ami="ami-0c7217cdde317cfec"             # AWS Ubuntu 22.04 LTS AMI for your region
+ec2_instance_size="t2.small"                # Instance Size for the machine running the catena backend
+ssh_private_key_path="~/.ssh/id_rsa"        # Your private key
+ssh_public_key_path="~/.ssh/id_rsa.pub"     # Your public key
+route_53_hosted_zone_name="catenatools.com" # The name of the hosted zone in route53
+letsencrypt_email="example@example.com"     # The email address LetsEncrypt will send cert expiry notifications to
+```
+
+### Provision the Infrastructure
+
+```bash
+terraform init                        # Initialize the terraform backend and install modules
+terraform workspace switch -or-create dev
+terraform plan -var-file="vars.tfvar" # Preview the steps terraform will take
+terraform apply -var-file="vars.tfvar"
+```
+
+After the apply step runs, you should see some output like this:
+
+```
+add_dokku_remote_command = "git remote add dokku dokku@dev.catenatools.com:platform"
+ec2_instance_id = "i-0f47e887113bf882b"
+ec2_instance_ssh_command = "aws ec2-instance-connect ssh --os-user ubuntu --instance-id i-0f47e887113bf882b --profile catena"
+ec2_ip = "100.28.26.117"
+powershell_deploy_command = "$env:GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa -o IdentitiesOnly=yes'; git push dokku main"
+unix_deploy_command = "GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa -o IdentitiesOnly=yes' git push dokku main"
+windows_deploy_command = "set \"GIT_SSH_COMMAND=ssh -i ~/.ssh/id_rsa -o IdentitiesOnly=yes\" && git push dokku main"
+catena_url = "https://platform.dev.catenatools.com"
+```
+
+These outputs will be used to deploy the platform to your newly configured instance.
+
+> The rest of this tutorial will use the environment variable `$CATENA_URL`. Set it to the value of the catena_url output.
+> ```bash
+> export CATENA_URL="https://platform.dev.catenatools.com"
+> ```
+
+### Install Catena
+
+Installing Catena is as easy as doing a git push.
+
+Start by cloning the `catena-tools-core` repo:
+
+**Git Bash**
+```bash
+git clone git@github.com:CatenaTools/catena-tools-core.git
+cd catena-tools-core/
+```
+
+Then add a new remote and do a git push, this will deploy the `HEAD` of the repo. The command to do so is the value of the `add_doku_remote_command` output.
+
+**Git Bash**
+```bash
+git remote add dokku dokku@dev.catenatools.com:platform
+git push dokku HEAD:main
```

Comment:
If for some reason a developer uses a key _not_ called `id_rsa`, they'll need to use the relevant `deploy_command` from the Terraform output.